benchmarks/benchmark_beam_search_server.py

```diff
@@ -0,0 +1,277 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: Copyright contributors to the vLLM project
+"""Simple benchmark script to test beam search performance via OpenAI API.
```

Comment:
I think we have can benchmark beam search with vllm bench latency and vllm bench throughput

Reply:
I didn't realize, will try to use it